newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Related Tracks'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '978d9c26606ebb5950841347f30888339d6840c1',
```

Comment:
i'm not 100% sure but you may want to gitignore this file if there's a key in it